### DISCUSSION TITLE
The Coordinates Not Set Properly

### DISCUSSION THREAD

**Discussion by farasjibran** [MatthiasValvekens/pyHanko#342] [Q&A]
**Describe the bug**
I have problems when I want to sign using this library. When I try to sign with a PDF file that has a PDF version below 1.7, the signature I put does not match the location or coordinates I have provided.

**To Reproduce**
this my code to sign

`
fields.append_signature_field(
      w, 
      sig_field_spec=fields.SigFieldSpec(
          f'InternalSignature{order}', 
          box=(int(x1), int(y1), int(x2), int(y2)),
          on_page=(newPageNumber)
  ),
)
`

**Expected behavior**
My expectation is that I can sign with PDF files that have a PDF version below 1.7

**Screenshots**

This my pdf and the version of the pdf and this works :
<img width="899" alt="image" src="https://github.com/MatthiasValvekens/pyHanko/assets/50952131/d9b93d68-e979-44fb-81a5-32f7334e4e72">
<img width="241" alt="image" src="https://github.com/MatthiasValvekens/pyHanko/assets/50952131/e7cd0f6e-04ea-4b73-9c9e-008d45a0e79b">

This my pdf and the version of the pdf and this not works properly :
<img width="890" alt="image" src="https://github.com/MatthiasValvekens/pyHanko/assets/50952131/60353559-1800-48b6-8913-caadfafaf510">
<img width="256" alt="image" src="https://github.com/MatthiasValvekens/pyHanko/assets/50952131/a793c939-4ba3-495a-8e8b-6b90e5d3ecb6">

**Environment (please complete the following information):**
 - OS: macOS
 - Version 12.7

**Comment by MatthiasValvekens** [comment]
Is the bottom PDF rotated? If so, that's a known issue. See #266 for some pending work to fix it.

**Comment by farasjibran** [comment]
> Is the bottom PDF rotated? If so, that's a known issue. See #266 for some pending work to fix it.

the bottom pdf is not rotated but i think the resolution is the one of the problem

**Comment by MatthiasValvekens** [comment]
> the bottom pdf is not rotated but i think the resolution is the one of the problem

Well, if your page is smaller, then obviously the coordinates of your box are going to change accordingly :). That's not so much a resolution issue as it is a geometry issue. So I don't really understand what you mean by this.
